### ISSUE TITLE
Conda Fails with create_default_packages

### ISSUE THREAD

**Issue by quasiben** [conda/conda#3453]
When using `create_default_packages`, `conda create...` fails. 

## Sample condarc:

```
channels:
  - defaults

create_default_packages:
  - python
  - pip
```

## Command
`conda create -n test_conda_update python=2 numpy`

## Error
```
    Traceback (most recent call last):
      File "/opt/a/b/c/muunitnoc/anaconda/lib/python2.7/site-packages/conda/exceptions.py", line 472, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/opt/a/b/c/muunitnoc/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 144, in _main
        exit_code = args.func(args, p)
      File "/opt/a/b/c/muunitnoc/anaconda/lib/python2.7/site-packages/conda/cli/main_create.py", line 68, in execute
        install(args, parser, 'create')
      File "/opt/a/b/c/muunitnoc/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 171, in install
        default_packages.remove(default_pkg)
    AttributeError: 'tuple' object has no attribute 'remove'
```

**Comment by kalefranz** [comment]
Fixed with #3454. Will be included in 4.2.8.  No timeline yet for that release.

**Comment by quasiben** [comment]
Any possibility of a release to conda-canary ?
